src/main/java/gov/usgs/wma/waterdata/timeseries/TimeSeriesDao.java

```diff
@@ -8,13 +11,25 @@
 @Repository
 public class TimeSeriesDao extends SqlSessionDaoSupport {
 
+	private static final String COLLECTION_KEY = "collectionId"; // network ID
+	private static final String FEATURE_KEY    = "featureId";    // location ID
+	private static final String TIMESERIES_KEY = "timeseriesId"; // GUID
+
 	@Autowired
 	public TimeSeriesDao(SqlSessionFactory sqlSessionFactory) {
 		setSqlSessionFactory(sqlSessionFactory);
 	}
 
-	public String getTimeSeries(String monitoringLocationId, String timeSeriesId) {
+	public String getTimeSeries(String timeSeriesId) {
 		return getSqlSession().selectOne("groundwaterDailyValue.getGeoJson", timeSeriesId);
 	}
 
+	public String getTimeSeries(String collectionId, String featureId, String timeSeriesId) {
+		Map<String,String> params = new HashMap<>();
+		params.put(COLLECTION_KEY, collectionId);
+		params.put(FEATURE_KEY, featureId);
+		params.put(TIMESERIES_KEY, timeSeriesId);
+		return getSqlSession().selectOne("timeseries.getGeoJson", params);
+	}
+
```

Comment:
It looks like the all  params.put calls are storing the same value: collectionId

Reply:
yep, I caught that too. Has been fixed.

Reply:
Never mind, seems I was looking at an old version.